src/autotester/server/utils/tests/file_management_test.py

```diff
@@ -0,0 +1,221 @@
+from autotester.server.utils.file_management import (
+    clean_dir_name,
+    random_tmpfile_name,
+    recursive_iglob,
+    copy_tree,
+    copy_test_script_files,
+    ignore_missing_dir_error,
+    move_tree,
+    fd_open,
+    fd_lock,
+)
+from autotester.server.utils import redis_management
+import os.path
+import re
+import tempfile
+import shutil
+from autotester.config import config
+import subprocess
+import pytest
+
+FILES_DIRNAME = config["_workspace_contents", "_files_dir"]
+
+@pytest.fixture
+def input1():
+    dir1 = tempfile.mkdtemp()
```

Comment:
same comment as before about using mkdtemp and mkstemp when you could be using TemporaryDirectory and NamedTemporaryFile